ReactWindows/ReactNative/Modules/DeviceInfo/DeviceInfoModule.cs

```diff
@@ -183,29 +148,58 @@ private void OnOrientationChanged(DisplayInformation displayInformation, object
         private void SendUpdateDimensionsEvent()
         {
             Context.GetJavaScriptModule<RCTDeviceEventEmitter>()
-                .emit("didUpdateDimensions", GetDimensions());
+                .emit("didUpdateDimensions", GetAllDimensions());
         }
 
-        private static JObject GetDimensions()
+        private JObject GetAllDimensions()
         {
-            return GetDimensions(DisplayMetrics.GetForCurrentView());
+            var dimensions = new JObject();
+
+            // Default metric for main window - is always first registered
```

Comment:
>is always first registered [](start = 48, length = 26)

Not really true for cases like "My People", "Share Panel" etc. Those start first, and even though main view (in UWP parlance) exists, it has no CoreWindow, and no react root views of course.

Reply:
Yeah, you are right, I missed that. I'll update the logic so the default will be always dimensions of Main window and Empty if CoreWindow doesn't exist - same logic as before.

Reply:
Fixed, please have a look